##CLOUD-SQL-CONFIGURATIONS
spring.cloud.gcp.sql.database-name=doconnectdb
spring.cloud.gcp.sql.instance-connection-name=carbide-bongo-406810:us-central1:myinstance
# So app starts despite "table already exists" errors.
spring.datasource.continue-on-error=true
# Enforces database initialization
spring.datasource.initialization-mode=always
spring.datasource.username=ranjith
spring.datasource.password=12345
spring.cloud.gcp.project-id=carbide-bongo-406810
spring.cloud.gcp.credentials.location=classpath:carbide-bongo-406810-a68f470b45a3.json

spring.jpa.hibernate.ddl-auto= update
spring.jpa.show-sql= true

spring.jpa.properties.hibernate.hbm2ddl.auto: update